lua/telescope/builtin/git.lua

```diff
@@ -107,37 +107,35 @@ end
 
 git.status = function(opts)
   local output = vim.split(utils.get_os_command_output('git status -s'), '\n')
-  local results = {}
-  for _, v in ipairs(output) do
-    if v ~= "" then
-      local mod, fname = string.match(v, '(..)%s(.+)')
-      if mod ~= 'A ' and mod ~= 'M ' and mod ~= 'R ' and mod ~= 'D ' then
-        table.insert(results, { mod = mod, file = fname })
-      end
-    end
-  end
 
-  if vim.tbl_isempty(results) then
+  if vim.tbl_isempty(output) then
```

Comment:
This check fails, i can't find the syntax to handle the empty result right. Any ideas? with vim.inspect, output is { "" }

Reply:
Okay i see. Because of the newline the terminal prints. Vim split will create an entry for each new line and this will include that one. I think we should just be able to get the output do a string check for `""` and than split